modules/face/src/face_alignment.cpp

```diff
@@ -0,0 +1,906 @@
+Copyright (C) 2010-2013, Advanced Micro Devices, Inc., all rights reserved.
+Copyright (C) 2015, OpenCV Foundation, all rights reserved.
+Copyright (C) 2015, Itseez Inc., all rights reserved.
+Third party copyrights are property of their respective owners.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+  * Redistributions of source code must retain the above copyright notice,
+    this list of conditions and the following disclaimer.
+
+  * Redistributions in binary form must reproduce the above copyright notice,
+    this list of conditions and the following disclaimer in the documentation
+    and/or other materials provided with the distribution.
+
+  * Neither the names of the copyright holders nor the names of the contributors
+    may be used to endorse or promote products derived from this software
+    without specific prior written permission.
+
+This software is provided by the copyright holders and contributors "as is" and
+any express or implied warranties, including, but not limited to, the implied
+warranties of merchantability and fitness for a particular purpose are disclaimed.
+In no event shall copyright holders or contributors be liable for any direct,
+indirect, incidental, special, exemplary, or consequential damages
+(including, but not limited to, procurement of substitute goods or services;
+loss of use, data, or profits; or business interruption) however caused
+and on any theory of liability, whether in contract, strict liability,
+or tort (including negligence or otherwise) arising in any way out of
+the use of this software, even if advised of the possibility of such damage.
+*/
+
+#include "opencv2/face/face_alignment.hpp"
+#include "opencv2/core/mat.hpp"
+#include <opencv2/objdetect.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/core.hpp>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <map>
+#include <vector>
+#include <cmath>
+#include <vector>
+#include <queue>
+#include <algorithm>
+
+using namespace std;
+
+namespace cv{
+namespace face{
+//! An enum which decides the type of cascade classifier chosen for face detection.
+/*! If 0 is chosen then HAAR cascade classifier is selected else LBP cascade classifier
+*is selected.
+*/
+enum face_cascade_classifier{ FACE_HAAR = 0, FACE_LBP = 1 };
+const string meanshapefile ="opencv2/modules/face/data/mean_shape.txt";
+/**@brief structure determining split in regression tree
+*/
+struct splitr{
+        //!index1 Index of the first coordinates among the test coordinates for deciding split.
+        unsigned long index1;
+        //! index2 index of the second coordinate among the test coordinates for deciding split.
+        unsigned long index2;
+        //! thresh threshold for deciding the split.
+        float thresh;
+};
+/** @brief regression tree structure. Each leaf node is a vector storing residual shape.
```

Comment:
i guess you go the concept of the documentation wrong. Everything gets processed from header files. Documentation in cpp source files is never processed into documentation, so it should not be there.